spec/models/state_file_base_intake_spec.rb

```diff
@@ -105,6 +105,18 @@
 
       expect(w2.employer_state_id_num).to eq "000000005"
     end
+
+    it "adds box 14 fields" do
```

Comment:
oh i left a comment on the controller spec about validations: any specs for validations should go here, not in the controller spec, oops